recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,169 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
+  #url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
+  md5: 6e27c7b5c1398bc56156ba9d56b43679
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - runbenders=pyomo.pysp.benders:Benders_main
+    - evaluate_xhat=pyomo.pysp.evaluate_xhat:EvaluateXhat_main
+    - runph=pyomo.pysp.phinit:PH_main
+    - runef=pyomo.pysp.ef_writer_script:main
+    - pysp2smps=pyomo.pysp.smps.pysp2smps:main
+    - phsolverserver=pyomo.pysp.phsolverserver:main
+    - scenariotreeserver=pyomo.pysp.scenariotree.server_pyro:main
+    - computeconf=pyomo.pysp.computeconf:main
+    - results_schema=pyomo.scripting.commands:results_schema
+    - pyro_mip_server=pyomo.scripting.pyro_mip_server:main
+    - test.pyomo=pyomo.scripting.runtests:runPyomoTests
+    - pyomo=pyomo.scripting.pyomo_main:main
+    - pyomo_ns=pyomo.scripting.commands:pyomo_ns
+    - pyomo_nsc=pyomo.scripting.commands:pyomo_nsc
+    - kill_pyro_mip_servers=pyomo.scripting.commands:kill_pyro_mip_servers
+    - launch_pyro_mip_servers=pyomo.scripting.commands:launch_pyro_mip_servers
+    - readsol=pyomo.scripting.commands:readsol
+    - ossolverservice=pyomo.scripting.commands:OSSolverService
+    - pyomo_python=pyomo.scripting.commands:pyomo_python
+    - pyomo_old=pyomo.scripting.pyomo_command:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyutilib >=5.3.5
+    - appdirs
+    - ply
+    - nose
```

Comment:
No, nose is not a build requirement.  I'll fix that.

On Wed, Apr 27, 2016 at 7:26 AM, Filipe <notifications@github.com> wrote:

> In recipes/pyomo/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/449#discussion_r61241849>
> :
>
> > +    - pyomo_nsc=pyomo.scripting.commands:pyomo_nsc
> > +    - kill_pyro_mip_servers=pyomo.scripting.commands:kill_pyro_mip_servers
> > +    - launch_pyro_mip_servers=pyomo.scripting.commands:launch_pyro_mip_servers
> > +    - readsol=pyomo.scripting.commands:readsol
> > +    - ossolverservice=pyomo.scripting.commands:OSSolverService
> > +    - pyomo_python=pyomo.scripting.commands:pyomo_python
> > +    - pyomo_old=pyomo.scripting.pyomo_command:main
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - setuptools
> > +    - pyutilib >=5.3.5
> > +    - appdirs
> > +    - ply
> > +    - nose
>
> Is nose a build requirement?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/449/files/ebc47763a7a7e7672134bb0b59d5f4486184a438#r61241849>
>